### DISCUSSION TITLE
How to set window position?

### DISCUSSION THREAD

**Discussion by ales-tsurko** [slint-ui/slint#3560] [Q&A] [answered]
I found API to do this from Rust, but in Slint language Window doesn't have `position` property. Setting `absolute-position`, `x` and/or `y` also doesn't have any effects.

**Comment by ogoffart** [comment, marked as answer]
It is not possible to set the Window position directly in .slint
You need to use the native API for that.

The rationale is that the Slint code should mainly be there for doing the design of the UI, and setting the position is not something a designer would design, it is more part of the logic.

A common use case i've seen for setting the position of a Window is to implement drag and move of the window. IMHO that is a valid use case and we should address it by having drag handle as tracked in https://github.com/slint-ui/slint/issues/613
